### ISSUE TITLE
Contributor Profiles: '+' and '-' seem a bit wonky

### ISSUE THREAD

**Issue by GuiltyDolphin** [duckduckgo/community-platform#1381]
See

![screenshot-2016-06-01-11 24 08_40_36](https://cloud.githubusercontent.com/assets/8598426/15706451/8bb32f3c-27eb-11e6-9c39-0511577ce428.png)

and

![screenshot-2016-06-01-11 24 19_42_40](https://cloud.githubusercontent.com/assets/8598426/15706457/9236eb6e-27eb-11e6-81c6-b96491680011.png)

@tagawa I believe you had a suggestion for this?

/cc @alohaas

**Comment by tagawa** [comment]
Yes, increasing the font size for the plus and minus symbols by 2px seems to work - 14px if I remember correctly.

**Comment by alohaas** [comment]
@jagtalon can you look into this?

**Comment by jagtalon** [comment]
@alohaas @moollaza thank you! Looking into it
